shared/oval/package_audit_installed.xml

```diff
@@ -5,11 +5,14 @@
     <metadata>
       <title>Package audit Installed</title>
       <affected family="unix">
-        <platform>multi_platform_rhel</platform>
+        <platform>CentOS 4</platform>
+        <platform>CentOS 5</platform>
```

Comment:
@shawndwells 

Hi Shawn,

> Asked a different way: On lines 10-12 you have added CPE tags for RHEL4, RHEL5,

RHEL4 and RHEL5 CPE tags are present in current RHEL/5 version of that OVAL.
> and RHEL6.

RHEL6 was included in current "multi_platform_rhel" which is in shared/ OVAL.
> On lines 8 and 9, you have added CentOS 4 and CentOS 5.

Same case like above, CentOS 4 and CentOS 5 are already present in:
  https://github.com/OpenSCAP/scap-security-guide/blob/master/RHEL/5/input/oval/package_audit_installed.xml#L8

This change is just moving those platforms from ```RHEL/5/input/oval/package_audit_installed.xml``` to ```shared/oval/package_audit_installed.xml```.

> Is there a reason RHEL6 was added,

RHEL6 is already present in shared version of the OVAL (covered by ```multi_platform_rhel```):
  https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/oval/package_audit_installed.xml#L8

```multi_platform_rhel``` = RHEL6, RHEL7 from config/oval.config:
https://github.com/OpenSCAP/scap-security-guide/blob/master/config/oval.config#L27

So this change is just:
* moving <platforms> from RHEL-5 script into shared/ one,
* keeping just RHEL-6 one in the shared/
* removing RHEL-7 one and moving it into shared/oval/oval-5.11 (the reason is we don't want ```package_audit_installed``` OVAL to be included on RHEL-7 always. Only in case the content is built on system supporting OVAL-5.11 (because ```package_audit_installed``` is just prerequisite for ```service_auditd_enabled``` OVAL check. Or vice versa, if ```service_auditd_enabled``` isn't included [it's not because systemdunitdependency -- OVAL 5.11 construct won't be included], it doesn't make sense to include ```package_audit_installed``` in that case too, because that would be unused OVAL. and ```verify-references.py``` would complain about that => ```make validate``` would fail).

> but not CentOS 6? Should we take this PR as an opportunity to add CentOS6 CPE tags?

No. Sorry. I am not willing to add ```CentOS 6``` tags via this PR. It's a blocker (till this isn't merged, all other PRs will be failing => people won't even consider reviewing them). Besides that adding ```CentOS 6``` tags would need further testing, if this doesn't break something else. I would rather have a separate PR doing nothing else, just adding ```CentOS6``` platform tags, than to combine them with this PR.

Sounds reasonable?

Thanks, Jan.